src/api/apple-game.api.ts

```diff
@@ -1,8 +1,6 @@
-import {
-  goldModeType,
-  checkMoveType,
-  gameEndType,
-} from '@pages/games/SnackGame/game/legacy/legacyType';
+
+
+import { checkMoveType, gameEndType, goldModeType } from '@pages/games/AppleGame/appleGameType';
 
```

Comment:
포맷팅 적용이 안 먹히고 있는 거 같은데 맞나요??

```suggestion
import {
  checkMoveType,
  gameEndType,
  goldModeType,
} from '@pages/games/AppleGame/appleGameType';

```